telemetry/client.go

```diff
@@ -0,0 +1 @@
+package telemetry
```

Comment:
why this  file is blank, if not required remove it.

Reply:
Done